src/storage/src/mem_table.rs

```diff
@@ -144,6 +161,7 @@ impl MemTable {
                             new: KeyOp::Delete(old_value),
                         }));
                     }
+                    self.kv_size.add_val(&original_new_value);
```

Comment:
Did we miss to subtract the original_new_value and original_old_value size here?

Reply:
Fixed.